obj2mjcf/_cli.py

```diff
@@ -9,15 +9,16 @@
 import tempfile
 from dataclasses import dataclass, field
 from pathlib import Path
-from typing import List, Optional, Sequence
+from typing import List, Optional
 
-import mujoco
 import trimesh
 import tyro
-from lxml import etree
 from PIL import Image
 from termcolor import cprint
 
+from obj2mjcf.Material import _MTL_COMMENT_CHAR, Material
+from obj2mjcf.MJCFBuilder import MJCFBuilder
```

Comment:
Switch to `from .material` and `from .mjcf_builder`.

Reply:
Sorry for the noob mistake. Just updated this line.